docs/development_guide.md

```diff
@@ -13,6 +13,13 @@ When this command is called, an instance of `Iniator` is created and the command
 All tests are run from the `tests` folder within the correlating command folder. They also run in the CircleCI build.
 Also make sure your methods work from the CLI by running `python demisto_sdk <your_method>` in your local environment.
 
+### How to run your unreleased demisto-sdk branch locally
```

Comment:
I think you've confused with the descriptions, pip install will update your env and tox will let you work in a virtual env. Am I wrong?

Reply:
You can run either of the commands. no use in running both.